lib/tasks/stop_data_convert_and_import.rake

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+namespace :bus_stops do
+  task import_old_data: :environment do
+    csv = CSV.read('old_stop_data.csv', headers: true)
+    csv.each do |row|
+      new_attributes = {}
+      new_attributes['has_power'] = case row['has_power']
+                                    when 'true' then 'Yes - Stub up'
+                                    when 'false' then 'No'
+                                    end
+      new_attributes['shared_sign_post_frta'] = case row['shared_sign_post']
+                                                when nil then nil
+                                                when 'Yes - FRTA' then true
+                                                else false
+                                                end
+      new_attributes['system_map_exists'] = case row['system_map_exists']
+                                            when 'true' then 'Old map'
+                                            when 'false' then 'No map'
+                                            end
+      new_attributes['trash'] = case row['trash']
+                                when nil then nil
+                                when 'None' then false
+                                else true
+                                end
+      stop = BusStop.find_by_id(row['id'])
```

Comment:
We should avoid the `find_by_xxx` in Rails >3. I would prefer `find_by id: row['id']` here, though bear in mind that's the same as `find row['id']`.